docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -112,6 +112,24 @@ where :math:`p(\text{RDS})` is the probability of dying from Preterm with RDS in
 
 When we fill in the location-specific values for delivery facility rates, CPAP coverage, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_RDS_w and p_RDS_wo), which we can solve analytically.
 
+We can derive the PAF and RR of lack of access to CPAP from these quantities as follows::
```

Comment:
@zmbc : Does this give enough context?  Maybe you can suggest an edit if it is not clear.

```suggestion
As mentioned above, we can calculate a PAF for "lack of access to CPAP".  This is convenient because Vivarium Public Health includes RiskExposure and RiskEffect components that can represent the way CPAP changes RDS mortality without requiring additional code.

We can derive the PAF and RR of "lack of access to CPAP" from these quantities as follows::
```

Reply:
Hm I want to suggest an edit but I'd like to edit line 65 as well and can't because you haven't touched it in this PR 🤔

OK I make a sub-PR into this branch?

Reply:
I'll merge this suggestion and you can make a PR onto my PR